client/src/components/dashboard/Dashboard.js

```diff
@@ -21,10 +21,13 @@ import Webpages from './webpage/Webpages';
 import DeployPage from './deploy/DeployPage';
 
 const Dashboard = () => {
+  console.log('dashborad');

```

Comment:
same: prefer not to use logs like this in main, only for development

Reply:
removed